### PR TITLE
Smooch

### PR DESCRIPTION
**Pull request by Mario54**

### CODE REVIEW

File: source/includes/_adding_smooch_ios.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+![Dependencies in XCode](/images/dependencies.png)
+
+Now you'll have to add Smooch's dependencies to your project if they're not already linked in. Go to "Build phases" in your project's target and select "Link Binary With Libraries":
+
+ * CoreGraphics.framework
+ * CoreTelephony.framework
+ * CoreText.framework
+ * Foundation.framework
+ * libxml2.dylib
```
Comment (mspensieri) [inline source/includes/_adding_smooch_ios.md]: libxml2 and openGLES are no longer required frameworks

File: source/includes/_adding_smooch_ios.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+To update via cocoapods, simply execute this
+
+```
+$ pod update
+```
+
+### Manual
+
+To manually update Smooch simply grab a fresh copy of Smooch by [downloading the most recent release](https://github.com/radialpoint/Smooch/archive/master.zip) and re-add the SDK to your XCode project by dragging both Smooch.framework and Smooch.bundle into your project
```
Comment (mspensieri) [inline source/includes/_adding_smooch_ios.md]: https://github.com/radialpoint/Smooch/archive/master.zip -> https://github.com/smooch/smooch-ios/archive/master.zip

File: source/includes/_localization.md [hunk 3]
```diff
@@ -16,6 +16,6 @@ Localization is subject to caching. If you can't see your changes, cleaning your
...
+To enable other languages beside the provided ones, first copy the english SmoochLocalizable.strings file from the Smooch bundle to the corresponding .lproj folder for that language. Then, translate the values to match that language.
 
-If you translate SupportKit's strings to a language not currently supported, we encourage you to share it with us so that every SupportKit user can benefit from it. You can do so by [forking](https://github.com/radialpoint/SupportKit/fork) our GitHub repo and creating a pull request, or just send us your string file at <a href="mailto:help@supportkit.io">help@supportkit.io</a>
+If you translate Smooch's strings to a language not currently supported, we encourage you to share it with us so that every Smooch user can benefit from it. You can do so by [forking](https://github.com/radialpoint/Smooch/fork) our GitHub repo and creating a pull request, or just send us your string file at <a href="mailto:help@smooch.io">help@smooch.io</a>
```
Comment (mspensieri) [inline source/includes/_localization.md]: https://github.com/radialpoint/Smooch/fork -> https://github.com/smooch/smooch-ios/fork

File: source/includes/_recommending_answers.md [hunk 4]
```diff
@@ -1,31 +1,32 @@
+
 # Recommending answers
```
Comment (mspensieri) [inline source/includes/_recommending_answers.md]: Recommending answers is no longer a part of the smooch core sdk. Not sure what to do with this section

File: source/includes/_swipe_gesture.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 # Swipe gesture for help
```
Comment (mspensieri) [inline source/includes/_swipe_gesture.md]: This too, no longer available in the core sdk

File: source/includes/integrations/_knowledgebase_ios.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ### Configuring Zendesk knowledge base
```
Comment (mspensieri) [inline source/includes/integrations/_knowledgebase_ios.md]: Ditto. no more KB integration

File: source/includes/_adding_smooch_ios.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+```
+
+Open the .xcworkspace file and add the necessary code to [initialize Smooch in your app](#import-the-smooch-header-file). 
+
+<aside class="notice">
+Remember — when you use CocoaPods to manage your app's dependencies, you have to build and run your app using the .xcworkspace file and not the .xcodeproj file.
+</aside>
+
+## Manual Method
+First, grab a copy of Smooch by [downloading the most recent release](https://github.com/radialpoint/Smooch/archive/master.zip) or visiting our [GitHub page](https://github.com/radialpoint/Smooch).
```
Comment (mspensieri) [inline source/includes/_adding_smooch_ios.md]: radialpoint/Smooch -> smooch/smooch-ios

**Comment by jugarrit** [comment]
Looks gouda to me
:+1:

File: source/includes/_adding_smooch_ios.md (hunk 1 above)
Comment (Mario54) [inline source/includes/_adding_smooch_ios.md]: Remove this too, then?

```
<aside class="warning">
Starting in Xcode 7, `libxml2.dylib` is no longer available in the frameworks list. Instead, you need to add the `-lxml2` option to your app's `Other Linker Flags` build setting.
</aside>
```

Comment (mspensieri) [inline source/includes/_adding_smooch_ios.md]: Yep

**Comment by mspensieri** [comment]
:+1:

**Comment by dannytranlx** [comment]
:+1:

**Comment by jpjoyal** [comment]
:+1:

**Pull request merged**